app/assets/stylesheets/pages/_roasts.scss

```diff
@@ -0,0 +1,256 @@
+$range-track-color: #DBDBDB;
+$range-handle-color-center: #FF6363;
+$range-handle-size: 36px;
+$range-handle-size-big: 67px;
+$range-handle-size-center: 0.45vw;
+
+@mixin flex-centered {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.left-axis {
+  @include fluid-sizeFULL('left', 0px, 140px);
+}
+
+.right-axis {
+  @include fluid-sizeFULL('right', 0px, 140px);
+}
+
+.roasts {
+  display: flex;
+  flex-direction: column;
+  .roasts__slider-wrapper {
+    display: flex;
+    flex-direction: column;
+    order: 2;
+    align-self: center;
+    margin-top: 10%;
+    margin-bottom: 10%;
+    width: 100%;
+    align-items: center;
+    .slider-axis {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 61%;
+      margin-bottom: 3%;
+      @include breakpoint(300px 1000px) {
+        justify-content: space-around;
+        margin-bottom: 4%;
+      }
+      @include breakpoint($large-screens) {
+        width: 100%;
+      }
+      span {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
```

Comment:
Is this `space-evenly` a firefox fix? I'm seeing on some random articles that this might not have full support.